input/tangy-radio-buttons.js

```diff
@@ -147,6 +147,11 @@ class TangyRadioButtons extends PolymerElement {
         value: false,
         observer: 'reflect',
         reflecttoattribute: true
+      },
+      correctValue: {
+        type: String,
+        value: '',
+        reflectToAttribute: true
```

Comment:
We could leave this out as it is out of the concern of TangyRadioButtons. The trick is it then does not get picked up in `TangyRadioButtons.getProps()` or avialbe via `TangyRadioButtons.correctValue` property. You have to `TangyRadioButtons.getAttribute('correct-value')` in situations like in editor where you upcast from DOM to JSON.

Reply:
Upon further reflection, `correct-value` may be in the concern of `TangyRadioButtons` which would allow it to determine if it is indeed correct while `incorrect-threshold` is a concern of `TangyFormItem` who can then ask it's children, "Are you correct?". This correct/incorrect pattern could follow how `invalid` works. You run `SomeInput.validate()` on an input and if it is invalid, an `invalid` property pops up on the element. In addition to potentially adding an `invalid` property to itself during `validate()` it could also determine if it needs to add an `incorrect` attribute. For some kinds of inputs such as a single select style of input, it can determine if the `incorrect`attribute should be applied immediately. However, with things that sometimes require multiple selections to be correct such as a `<tangy-checkboxes>` or `<tangy-eftouch>`, application of the `incorrect` attribute has to wait until the `validate` function is called.

Note this is not how `tangy-eftouch` currently works but I like the idea of figuring out a generalized pattern so we don't head down the `incorrect` path ;). Seriously though, if this does the job it needs to and we're out of time, maybe it's fine for now.